hazelcast/src/main/java/com/hazelcast/cache/CacheMergePolicy.java

```diff
@@ -21,36 +21,36 @@
 import java.io.Serializable;
 
 /**
- * <p>
- * Policy for merging cache entries.
- * </p>
- *
+ * Policy for merging cache entries after a split-brain has been healed.
  * <p>
  * Passed {@link CacheEntryView} instances wraps the key and value as their original types
- * with convertion to object from their storage types. If user doesn't need to original types of key and value,
- * (s)he should use {@link StorageTypeAwareCacheMergePolicy} which is sub-type of this interface.
- * </p>
+ * with conversion to object from their storage types. If user doesn't need to original types of key and value,
```

Comment:
.....storage types. If the user doesn't need the original.....
or
.....storage types. If you don't need the original types of key and value, you should use......